### DISCUSSION TITLE
Modify Code for Folder Upload

### DISCUSSION THREAD

**Discussion by jinruiiii** [ckan/ckan#7676] [General]
I'm currently working on a project where I need to enable folder-level uploads in CKAN's resource page. I would like to allow users to upload entire folders containing multiple files instead of selecting individual files one by one. Could anyone guide me on how to approach this?

**Comment by ghy99** [comment]
That is a very good question. Any solutions? I am encountering the same issue.

**Comment by hyhyzxc** [comment]
Amazing question!!! Great insightful question i am so amazed uwu uwu xdxd hehexd

**Comment by Pooya-Oladazimi** [comment]
We developed a plugin for our project to enable users to upload multiple data resources for a dataset. It does not support what you are looking for exactly, but it may be a good start to look at it:
https://github.com/TIBHannover/ckanext-multiuploader
